docs/administer/rbac.md

```diff
@@ -0,0 +1,265 @@
+# Role Based Access Control
+
+Role-based access control (RBAC) restricts access to resources within Percona Everest. It establishes a framework that defines access permissions and privileges according to individual users' roles. With RBAC, only authorized individuals can access specific information or perform certain actions based on their assigned roles. This method improves security by minimizing the risk of unauthorized access and helps manage permissions more efficiently across Percona Everest.
+
+
+## Default role
+
+In Percona Everest, the only predefined default role is **admin**. The **admin** user is a superuser and it has unrestricted access to Percona Everest. However, the RBAC (Role-Based Access Control) configuration can define and allocate specific roles based on individual requirements and access privileges.
+
+- `role:admin` - unrestricted access to all resources
+
+This default built-in role definition can be seen in [policy.csv](https://github.com/percona/everest/blob/main/deploy/quickstart-k8s.yaml#L91).
+
+## RBAC resources and privileges
+
+Below is a comprehensive table outlining the permissions granted for various **resources**:
+
+??? info "Permissions for resources"
+
+
+    !!! info "Important"
+        If you have permission for specific namespaces or resources, you can perform read, update, create, or delete actions only on those resources or only within those namespaces. However, if you have permission for **all** the resources or namespaces, you can carry out these actions across all the resources and namespaces.
+
+    </br> </br>
+
+    **Table: Privileges for the various resources in Percona Everest**
+
+    |**Resource**| **Read**| **Create**| **Update**|**Delete**|
+    |------------|---------|-----------|-----------|----------|
+    | Namespaces|You can view namespaces| :x:  |:x: | :x:|
+    | Database engines</br> MySQL, MongoDB, PostreSQL|You view can database engines when you create databases</br>  </br>**Note**: This policy must at least be **read all** so the users can create databases.| :x: |Modify database engines| :x:|
+    | Database clusters|You can view databases| You can create databases|You can modify databases| You can delete databases|
+    | Backups storage and Monitoring endpoints|You can siew backups and monitroing endpoints| You can create backups and monitroing endpoints|You can modify backups and monitroing endpoints| You can delete backups and monitroing endpoints|
+    | Database data (credentials)|View database data (credentials) </br> </br>**Note**: If no policy is defined:</br> * You cannot see the credentials and the connection string.</br> * You also cannot create a database from any backup.| :x: |:x:| :x:|
```

Comment:
I think in this table we should use the same name as we do in the RBAC policy. Also `database-cluster-backups` and `database-cluster-restores` are missing

```suggestion
    |**Resource**| **Read**| **Create**| **Update**|**Delete**|
    |------------|---------|-----------|-----------|----------|
    | namespaces|You can view namespaces| :x:  |:x: | :x:|
    | database-engines </br> (MySQL, MongoDB, PostreSQL)|You view can database engines when you create databases</br>  </br>**Note**: This policy must at least be **read all** so the users can create databases.| :x: |Modify database engines| :x:|
    | database-clusters|You can view databases| You can create databases|You can modify databases| You can delete databases|
    | backup-storages and monitoring-instances |You can siew backups and monitroing endpoints| You can create backups and monitroing endpoints|You can modify backups and monitroing endpoints| You can delete backups and monitroing endpoints|
    | database-cluster-credentials|View database data (credentials) </br> </br>**Note**: If no policy is defined:</br> * You cannot see the credentials and the connection string.</br> * You also cannot create a database from any backup.| :x: |:x:| :x:|
```